types/src/human_readable_compatibility.rs

```diff
@@ -0,0 +1,132 @@
+use crate::{contracts::*, Groups, URef};
+#[cfg(test)]
+use crate::{AccessRights, Group};
+use alloc::vec::Vec;
+#[cfg(test)]
+use alloc::{collections::BTreeSet, vec};
+use core::{convert::TryFrom, fmt};
+#[cfg(feature = "json-schema")]
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+#[cfg_attr(feature = "json-schema", derive(JsonSchema))]
+pub struct ContractVersionAndHash {
```

Comment:
I think this does not need to be `pub`. Consider checking visibility for other added items.

Reply:
no longer applicable